src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
 
 {{ figure_markup(
   image="seo-outgoing-links.png",
-  caption="Outgoing links",
-  description="Bar chart showing the number of outgoing links per percentile (10, 25, 50, 75, and 90). The median desktop homepage features 76 outgoing links while the median mobile homepage features 67.",
+  caption="Distribution of the number of outgoing links per page.",
+  description="Bar chart showing the number of outgoing links per percentile (10, 25, 50, 75, and 90). The median desktop home page features 76 outgoing links while the median mobile home page features 67.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=284216213&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median desktop page features 61 outgoing internal links while the median mobile page features 54 outgoing internal links. This is down 12.8% and 10% respectively from [last year’s analysis](../2019/seo#linking). This suggests that sites are not maximizing the ability to improve the crawlability and link equity flow through their pages in the way they did the year before.
+{# TODO(authors): Can you clarify how a link can be both outgoing and internal? #}
+The median page has 61 outgoing internal links on desktop and 54 on mobile. This is down 12.8% and 10% respectively from [last year's analysis](../2019/seo#linking). This suggests that sites are not maximizing the ability to improve the crawlability and link equity flow through their pages in the way they did the year before.
 
 {{ figure_markup(
   image="seo-outgoing-links-internal.png",
-  caption="Outgoing Links (Internal)",
+  caption="Distribution of the number of outgoing internal links per page.",
   description="Bar chart showing the number of internal outgoing links per percentile (10, 25, 50, 75, and 90). The median desktop page features 61 outgoing internal links while the median mobile page features 54 outgoing internal links.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=739265254&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median homepage is linking to external sites 7 times on desktop and 6 times on mobile.
+{# TODO(authors): Add your interpretation of these results. What do you hope readers will get out of it? If there's not much to say, would anyone miss it if it was dropped? Also, please clarify the redundancy of outgoing and external for a non-SEO like myself :) #}
```

Comment:
@aleyda

Reply:
@rviscomi I'll try to add more context here. Please note that "outgoing" and "external" are not redundancies. An outgoing link is a links that goes "out" from a page to another (rather than an incoming link, pointing to you). It's an "outgoing external" link when the link goes out to a page in another domain/site, and an "outgoing internal" going to another page belonging to the same domain/site. I hope this adds more context on why it's important to differentiate (linking to your internal pages vs. linking to other sites).

Reply:
Thanks you for clarifying. Can "outgoing" be assumed, or would it help to disclaim that at the top of this section, eg "We are only able to assess the outgoing links from each page and have not analyzed incoming links as part of this chapter." Then this section only needs to deal with internal vs external links, which I think everyone is familiar with but would still benefit from the short descriptions you've added.

Reply:
@rviscomi thanks for the clarification Rick! I just did the change by adding the context of "outgoing" in the intro to avoid mentioning them every single time. Also took the opportunity to add more context and links regarding links Google best practices. Thanks!